libethereum/EthereumPeer.cpp

```diff
@@ -37,8 +37,10 @@ using namespace dev;
 using namespace dev::eth;
 using namespace p2p;
 
-const unsigned c_maxIncomingNewHashes = 1024;
-const unsigned c_maxHashesToSend = 65536;
+static const unsigned c_maxIncomingNewHashes = 1024;
+static const unsigned c_maxHeadersToSend = 65536;
+static const unsigned c_maxBlockSkip = 99999;
```

Comment:
`skip` requires linear traversal by block headers currently. Big values would require to fetch whole db. Remove the limit?